utils/http/http.go

```diff
@@ -36,70 +32,3 @@ func GenBody(t interface{}) io.Reader {
 	log.Println("Request Body : ", string(tjson))
 	return bytes.NewReader(tjson)
 }
-
-// http post
-func PostRequest(ctx context.Context, transport http.RoundTripper, url string, body io.Reader) ([]byte, error) {
```

Comment:
why are we removing these functions?

Reply:
because these function are only used by plugin actually, and these func need to import lib from `github.paypal.com`, like monitoring lib, i think we need to move under `dce-plugin`, https://github.paypal.com/PaaS-R/dce-plugins/blob/2c482e0bc7deb5f11191c963a61a9d9829db8ca8/plugin/utils/http/http.go

Reply:
done with reverted the change